### DISCUSSION TITLE
Issues logging into my vercel account via github (sasaxton15)

### DISCUSSION THREAD

**Discussion by sasaxton15** [vercel/community#4130] [Account Issue]
### Question

I'm currently having issues logging into my Vercel account via github authentication. It looks like my user is block, is there a path forward to fix this issue?

**Comment by amyegan** [comment]
Sorry you're having trouble with your account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps.

For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)
